api/facadeversions.go

```diff
@@ -19,7 +19,7 @@ var facadeVersions = map[string]int{
 	"AllWatcher":                   1,
 	"Annotations":                  2,
 	"Application":                  5,
-	"ApplicationOffers":            1,
+	"ApplicationOffers":            2,
```

Comment:
We really really do not like bumping api versions in point release. I agree that this change warrants the version bump but does it really need to go into a pint release?

Reply:
So long as we are compatible with < 2.3.5 then I don't see an issue.
If's a usability issue we do need to address.